packages/common/src/components/utils/Modal/PopUpModal.tsx

```diff
@@ -1,19 +1,28 @@
 import React from 'react'
+import { CustomButton } from '../..'
 import { ModalBase } from '../..'
 
 interface Props {
   open: boolean
   modalTitle: string
+  handleClose: () => void
+  showCloseButton: boolean
 }
 
-const PopUpModal: React.FC<Props> = ({ open, modalTitle, children }) => {
+const PopUpModal: React.FC<Props> = ({ open, modalTitle, handleClose, showCloseButton, children }) => {
```

Comment:
Tests are missing for new functionality. Also, now that the PopUpModal can have buttons I don't see the difference between the ModalBase. It seems like this component is just wrapping the ModalBase.